docs/design-documents/docker_management/README.md

```diff
@@ -106,15 +106,13 @@ There are some details worth mentioning though.
 
 docker buildx command is used for creating multi architecture docker image. To build docker image using buildx, one needs to push the image to a remote repository while building it. Since, we need to "test" before release, the resulting images are pushed to a temporary docker repository just after building.
```

Comment:
```diff
- docker buildx command
+ The `docker buildx` command
```

Reply:
There are a few places where Docker is referring to the technology or company and not the command line command. Use "Docker" with a capital D for referring to the technology or company, and use `docker` when referring to the command line command.